databases/litecli/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        dbcli litecli 1.0.0 v
+revision            0
+
+categories          databases python
+maintainers
+license             BSD
+platforms           darwin
+supported_archs     noarch
+description         CLI for SQLite Databases with auto-completion and syntax highlighting
+long_description    ${description}
+
+homepage            https://litecli.com/
+
+checksums           rmd160  a3fc906e63f010eb92dd2bd5d55cdf7d12525a81 \
+                    sha256  59d652d868ab7525126c7c72b6a39259857f850fdbbe7e2b95c3f4dcc42f1601 \
+                    size    881824
+
+variant python35 conflicts python36 python37 description "Use Python 3.5" {}
+variant python36 conflicts python35 python37 description "Use Python 3.6" {}
+variant python37 conflicts python35 python36 description "Use Python 3.7" {}
+
+if {[variant_isset python35]} {
+    python.default_version 35
+} elseif {[variant_isset python36]} {
+    python.default_version 36
+} else {
+    default_variants +python37
+    python.default_version 37
+}
+
+depends_lib-append  port:py${python.version}-cli-helpers \
+                    port:py${python.version}-click \
+                    port:py${python.version}-configobj \
+                    port:py${python.version}-prompt_toolkit \
+                    port:py${python.version}-pygments \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-sqlparse
+
+    post-destroot {
+        set docdir ${prefix}/share/doc/${subport}
+        xinstall -d ${destroot}${docdir}
+        xinstall -m 0644 -W ${worksrcpath} LICENSE changelog.md ${destroot}${docdir}
+    }
```

Comment:
This block should not be indented. Also it references `${subport}` but it should be `${name}`.

Reply:
fixed, file was based on pgcli's Portfile and this is how it looks there